py/openage/convert/mediafile.py

```diff
@@ -77,6 +136,10 @@ def media_convert(args):
         args.extract.append('*:*.*')
 
     extraction_rules = [ExtractionRule(e) for e in args.extract]
+    if args.jobs == 0:
+        args.jobs = os.cpu_count()
```

Comment:
Could do, but I didn't want to add the `import os` in `__main__.py`. Also, in the first version, in this file the code was
```
if args.jobs == 0:
    pool = mp.Pool()
else:
    pool = mp.Pool(processes=args.jobs)
```
As `Pool`'s `processes` argument defaults to `os.cpu_count`, but it's not exposed later on and I wanted to let user know how many threads we use.